### DISCUSSION TITLE
Chia network growth. Calculation

### DISCUSSION THREAD

**Discussion by MrSosikin** [Chia-Network/chia-blockchain#6137] [Show and tell]
I have tried to calculate the possible chia`s network size and growth.

I thought that for the possible network size calculation it should take into account HDD disks manufactured at least since 2015 (even maybe older).

For that period it was manufactured:
2015 – 535,8 Exabyte
2016 -  633 Exabyte
2017 – roughly 768 Exabyte (didn’t find the accurate volume)
2018 – 800 Exabyte
2019 – roughly 900 Exabyte (didn’t find the accurate volume)
2020 – 1018 Exabyte

Total: 4654,8 Exabyte

We can also add about 100 Exabyte for cheap SSD which can also be used in farming.

So, it will be about 4800 Exabyte by the end of 2020.

From this number we can take away 60% for data centres and other businesses (of course they can be used for plotting and farming and in some limited quantities they are used for these purposes) and, maybe like nearly 30% for devices, which are installed in PCs and notebooks as a main drive or have a small capacity, or used like a media storage or are broken or doesn’t not suit for farming due to other reason (anyway it’s only my assumptions).

So, it will be left something like 50 Exabyte, and I presume that only 2-5% of this capacity or even less would be used in chia farming – not more than 2,5 Exabyte.

Of course, this calculation is extremely inaccurate but makes it possible to assume, that the chia network is growing thanks to mainly new devices. 

In first quarter 2021 it was sold nearly 344 Exabyte (288,3 Exabyte of HDD and 66 Exabyte of SSD) and to this date this number should have doubled.

In this way we have + 700 Exabyte, manufactured in the first half of the 2021 minus something like 80% for data centres and computer manufacturers orders. 

So, it will be roughly 140 Exabyte, which potentially could be used for chia farming..and till the end of the year it will be minimum + another 140 Exabyte (totally 280 Exabyte to the end of 2021).

Potentially it’s equal to average 1000 PB daily growth from the chia`s start to the end of 2021.

Thereby I think that it is no strong technical limitation for chia network growth and it has only economical limitation.

And about economical limitation. With current growth rate (more than 700 PiB per day for the last 10 days) it is needed nearly 505 plots or 55 TB to receive just 2 XCH during one year or to receive 2.67 XCH during 3 years (you can check it here https://chiacalculator.com/ with parameters "Network growth" 0 and "Stabilized daily growth" 700). It is nearly 1500-1800$ only for HDD (and as you may know without pools it’s a lottery) and a lot of time. 
So, for a reasonable person it is clear that it will be easier and cheaper to buy XCH for 830$. 

Therefore a lot of people will stop plotting soon because they will run out of free disk space and buying new discs will be inappropriate. 

On the other hand there are a lot of guys who have bought a suitable hardware and a huge amount of HDD volume and they can catch up or even overtake the growth of the network. They will continue plotting farther and I guess it is these people who are mainly adding the volume. 

So I think that maybe after one month the network growth rate will decrease because a lot of people will stop plotting but despite of it for several month the average chia network growth will be around 400-800 PB per day and to the end of 2021 the chia`s netspace will be around 100 - 160 Exabyte.

It is obvious that the further chia network growth will depend on XCH price and its expectations.

**Comment by tiny0little** [comment]
thx for sharing!!! please keep analyzing and updating us here

**Comment by chiabug** [comment]
I tend to agree with much of this information.  I believe that the calculations need to include disk estimates farther back than just 5 or 6 years.  A lot of people are competing to see how fast a plot can be made.  Most attempts follow the same path as other coin mining -- throw more and more new hardware at the problem.  This of course adds more cost to the equation.  I believe there is an underestimated number of users who are plotting on old hardware.  This type of plotting can be done for a fairly low cost but does require more patience.  Set up a few machines and forget them for a day or two.  When you come back they have sprouted 3 or 4 plots.      The Chia farming lottery really isn't a short term race that depends on speed as much as it is a long term business requiring sustainable management of a farm.

The k32 requirement also adds a wrinkle to the space calculations.  The total "sold" disk space of prior years has to be factored by the number of drive that were 250 GB or larger.  "Old hardware" farming with smaller drives is possible but gets more complicated.  Realistically 500 GB drives are near the low end of feasibility when using k32 plots.  Is there a breakdown of historical manufacturing of disk space by drive size?

As stated in the original post, participation in the Chia farming lottery may already be unprofitable for many farmers.  Additionally, the "technical" maintenance cost of many farms will rise as the farms get bigger  and more complex.  "Pools" may help keep some small farmers in the game because, theoretically, it will allow small farms to see some reward for their farming efforts versus the current "hope and pray" environment of the Chia lottery.

Having worked with many farmers I think another factor in the total space calculation is the number of "bad plots".  Currently there are many issues that can cause "bad plots".  I suspect this may be as much as 3 percent of the plots.  As the Chia software matures these plots, and the underlying causes, will be eliminated.  To date many small farmers think they have working plots and their farms are counted in the totals for the network.  In reality some of these plots are really not eligible to ever win a reward.  

As the network space becomes owned by more and more big farms with high speed connections the chance of a small farmer winning a block, and processing a transaction, decreases drastically.  Many farms are unaware that their connectivity, or the connectivity of their direct peers, limits their ability to ever win a block.  Software changes, and refinements in farming techniques, will help to eliminate some of the current hurdles.  Still the purposely limited rewards will continue to be shared by a bigger and bigger number of farmers. To be sustainable Chia needs to find a foothold in a sustained use market and get past the speculator "get rich quick" market.

**Comment by MrSosikin** [comment]
It's look like the stable growth has been achieved and it is nearly 700 PiB (or nearly 7 239 000 plots) per day. 

Also it's look like it started to decrease slowly (or maybe not, who knows..). 

From 05/23/2021 to 05/30/2021 the avarege growth was nearly 720 PiB / day, 
From 05/31/2021 to 06/07/2021 the avarege growth was nearly 680 PiB / day.

**Reply by MrSosikin** [reply]
Bram Cohen has commented that the drops in netspace are thanks to unofficial pools attemps to track down cheaters
![netspace-10062021](https://user-images.githubusercontent.com/84943548/121433374-9b1a7300-c984-11eb-9649-bad6aa6b26be.png)
![Bram_comment](https://user-images.githubusercontent.com/84943548/121433394-9f469080-c984-11eb-859f-d3223884ac02.png)

**Reply by dasrecord-zz** [reply]
![image](https://user-images.githubusercontent.com/50388266/155950638-7064e68c-13a7-4220-aba0-6e60b3b017bd.png)
